utils/filesystem.go

```diff
@@ -104,3 +111,81 @@ func PingHealth(healthEndpoint string) bool {
 	}
 	return started
 }
+
+//GetZipURL from github api /repos/:owner/:repo/:archive_format/:ref
```

Comment:
Could this comment be a bit more descriptive, if you're going to have it?
I also personally don't see the purpose of writing the function name in the comment, as it's obvious what it's for, is there a GO reason for that?

Reply:
Go linter. These are global functions so it wants/needs comments. I would prefer to keep these comments brief with just the key elements if possible - I dont think it really needs much more than this. Would you have a suggestion?

Reply:
`GetZipUrl from the github API, with format /repos/:owner/:repo/:archive_format/:ref` something like that? Comments are fine, I found this one not as clear as some of your others